src/image_deid_etl/image_deid_etl/main_pipeline.py

```diff
@@ -34,7 +34,7 @@ def subject_info(local_path, program, file_dir, validate=0):
         try:
             sub_mapping,sub_missing_c_ids,sub_missing_ses = get_subject_mapping_cbtn(cbtn_all_df,sub_info,local_path)
         except ValueError as error:
-            logger.error("Unable to get subject mapping without the images (program/site folder structure) present: %r", error)
+            logger.error("Error in getting CBTN subject mapping: %r", error)
```

Comment:
The error was too specific, it was being thrown even when the images were present. This error throws when the subject mapping can't be generated, but this could be due to multiple reasons - ideally there would be more discrete error handling which I will consider if it's difficult to debug.